index.js

```diff
@@ -81,6 +84,12 @@ function inject (bot) {
       const b = bot.blockAt(p)
       start = new Move(p.x, p.y + (b && dy > 0.001 && bot.entity.onGround && b.type !== 0 ? 1 : 0), p.z, movements.countScaffoldingItems(), 0)
     }
+    if (movements.allowEntityDetection) {
+      if (resetEntIntersects) {
+        stateMovements.entIntersections = {}
+      }
+      updateCollisionIndex(movements.entIntersections)
```

Comment:
Why is entIntersections not always cleared when updateCollisionIndex is called? updateCollisionIndex does not seam to ever remove or decrease any entries?

Reply:
Or is there some other place where entIntersections is cleared?

Reply:
Ah ok so entIntersections is cleared on every path reset. So does this mean that the entity map is always recalculated on path calculations? Why the resetEntIntersects check then?

Reply:
The main motivation for the `resetEntIntersects` option is so that a user can append their own index if they wanted to without it being cleared when the path is calculated. I am not too sure when this would be useful outside of tests.